src/EnergyPlus/ReportCoilSelection.cc

```diff
@@ -1907,7 +1907,7 @@ bool ReportCoilSelection::isCompTypeCoil(std::string const &compType // string c
 {
     // if compType name is one of the coil objects, then return true
     bool found(false);
-    for (int loop = 1; loop < DataHVACGlobals::NumAllCoilTypes; ++loop) {
+    for (int loop = 1; loop <= DataHVACGlobals::NumAllCoilTypes; ++loop) {
```

Comment:
This was skipping the last coil type in the list which is Coil:Cooling:DX:CurveFit:Speed.

Reply:
I do recall this.